response/slack/dialog_handlers.py

```diff
@@ -22,6 +22,11 @@ def report_incident(
     lead_id = submission["lead"]
     severity = submission["severity"]
 
+    if "incident_type" in submission:
+        report_only = submission["incident_type"] == "report"
+    else:
+        report_only = False
```

Comment:
This could be pushed down to the model (defaulting to False), but I have slight preference for it being explicitly set.